packages/fast-usdc/test/constants.test.ts

```diff
@@ -0,0 +1,12 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { TxStatus, PendingTxStatus } from '../src/constants.js';
+
+const { values } = Object;
+
+test('PendingTxStatus is a subset of TxStatus', t => {
+  const txStatuses = values(TxStatus);
+  const invalidStatuses = values(PendingTxStatus).find(
+    status => !txStatuses.includes(status),
+  );
+  t.is(invalidStatuses, undefined, `${invalidStatuses} not in TxStatus`);
```

Comment:
I was thrown off by the comparison with `undefined`.
```suggestion
  const allStatusus = new Set(values(TxStatus));
  const pendingStatuses = new Set(values(PendingTxStatus));
  const pendingMinusAll = pendingStatuses.difference(allStatuses);
  t.is(pendingMinusAll.size, 0);
```

Reply:
The main intention here was to ensure the extraneous value name(s) are in the error message. _TIL about `Set.difference`, pretty cool._ Unfortunately in our testing env, I'm seeing **pendingStatuses.difference is not a function**. I went with a slightly revised approach that's hopefully more readable: 9c14032354

Reply:
Oh, it's [not available until Node 22](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference#browser_compatibility). When we can use it, [isSubsetOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/isSubsetOf) would be even better